packages/client/src/pages/Safe.js

```diff
@@ -222,7 +184,11 @@ function Safe({ web3 }) {
 
   const onSend = () => {
     modalContext.openModal(
-      <SendTokens name={safeData.name} address={address} balance={balance} />
+      <SendTokens
+        name={safeData.name}
+        address={address}
+        allBalance={allBalance}
```

Comment:
Previously we are only storing the balance of the flow token, structure looking like 
```
{
   balances: {
        [treasuryAddress]: [flowBalance]
      }
}
```
 I changed it to
```
{
   balances: {
        [treasuryAddress]: {
              "FLOW": [flowBalance],
              "FUSD": [fusdBalance]
          }
      }
}
```
To better capture the idea that this field hold balance for all cointypes, I renamed it from balance to allBalance

Reply:
i see, but in `SendToken` seems we are using `balances` from context rather than the props passed here. https://github.com/DapperCollectives/VESSEL/pull/62/files#diff-f3affb1adea5ed2dfcb37ccdb85336346996959a2decd1882dd4aabd8c54a506R7

Reply:
CoinTypeDropDown reads the balances directly from the web3 context instead of the `allBalance` passed down the property chain. Since allBalance is not being used, I'm removing it from the sendToken property, thanks for raising the question! I definitely overlooked this part!